README.md

```diff
@@ -1,6 +1,6 @@
 <div align="center">
   <br>
-  <img alt="Open Sauced" src="https://i.ibb.co/7jPXt0Z/logo1-92f1a87f.png" width="300px">
+  <img alt="Open Sauced" src="https://github.com/open-sauced/assets/blob/main/logos/logo-on-dark.png" width="300px">
```

Comment:
Please revert this.

Reply:
I used “git revert” and it appears that it is reverted back now.  Look good?

Reply:
There is no new commit from you.

After you did `git revert`, you need to:

- add and commit your changes with `git add` and `git commit`
- push them to your remote branch with `git push`.